guides/common/modules/proc_configuring-ipxe-environment.adoc

```diff
@@ -0,0 +1,31 @@
+[id="Configuring_iPXE_Environment_{context}"]
+= Configuring iPXE environment
+
+Configure iPXE environment on all {SmartProxies} that you want to use for iPXE provisioning.
+
+ifndef::foreman-deb[]
+.HTTP security consideration
+In {EL}, security-related features of iPXE are not supported and the iPXE binary is built without security features.
```

Comment:
Does this relate to the OS of Foreman Server or the OS of your host?

Reply:
OS of Foreman Server/SmartProxy Server. Why?

Reply:
Was just wondering if the "ifdef" is correct.